Formula/tflint.rb

```diff
@@ -2,8 +2,8 @@ class Tflint < Formula
   desc "Linter for Terraform files"
   homepage "https://github.com/wata727/tflint"
   url "https://github.com/wata727/tflint.git",
-    :tag      => "v0.15.0",
-    :revision => "869c0a04c14ac316af0153ebc665b8a41e17ece5"
+    :tag      => "0.15.1",
```

Comment:
The tag is `v0.15.1`, with the leading `v`: https://github.com/terraform-linters/tflint/releases/tag/v0.15.1
```suggestion
    :tag      => "v0.15.1",
```